atoma-bin/atoma_node.rs

```diff
@@ -227,17 +227,21 @@ async fn main() -> Result<()> {
 
     let (compute_shared_secret_sender, compute_shared_secret_receiver) =
         tokio::sync::mpsc::unbounded_channel();
-    let confidential_compute_service = AtomaConfidentialComputeService::new(
-        client.clone(),
-        subscriber_confidential_compute_receiver,
-        app_state_decryption_receiver,
-        app_state_encryption_receiver,
-        compute_shared_secret_receiver,
-        shutdown_receiver.clone(),
-    )?;
 
+    let client_clone = client.clone();
+    let shutdown_receiver_clone = shutdown_receiver.clone();
     spawn_with_shutdown(
-        async move { confidential_compute_service.run().await },
+        async move {
```

Comment:
we don't need the async move, remove that and then you can get rid of the two clones and do them inline

Reply:
Oh nice ! Thanks for the insight